src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/SvDiscoverFromLocalAssemblyContigAlignmentsSpark.java

```diff
@@ -214,24 +217,69 @@ public static void dispatchJobs(final EnumMap<RawTypes, JavaRDD<AssemblyContigWi
 
         final String outputDir = svDiscoveryInputData.outputPath;
 
-        // TODO: 1/10/18 bring back imprecise variants calling (definitely not here) and read annotation
-        svDiscoveryInputData.updateOutputPath(outputDir+"/"+RawTypes.InsDel.name()+".vcf");
-        new InsDelVariantDetector()
-                .inferSvAndWriteVCF(contigsByPossibleRawTypes.get(RawTypes.InsDel), svDiscoveryInputData);
-
-        svDiscoveryInputData.updateOutputPath(outputDir+"/"+RawTypes.IntraChrStrandSwitch.name()+".vcf");
-        new SimpleStrandSwitchVariantDetector()
-                .inferSvAndWriteVCF(contigsByPossibleRawTypes.get(RawTypes.IntraChrStrandSwitch), svDiscoveryInputData);
-
-        svDiscoveryInputData.updateOutputPath(outputDir+"/"+RawTypes.MappedInsertionBkpt.name()+".vcf");
-        new SuspectedTransLocDetector()
-                .inferSvAndWriteVCF(contigsByPossibleRawTypes.get(RawTypes.MappedInsertionBkpt), svDiscoveryInputData);
+        // TODO: 1/10/18 bring back read annotation, see ticket 4228
+        forNonComplexVariants(contigsByPossibleRawTypes, svDiscoveryInputData);
 
         svDiscoveryInputData.updateOutputPath(outputDir+"/"+RawTypes.Cpx.name()+".vcf");
         new CpxVariantDetector()
                 .inferSvAndWriteVCF(contigsByPossibleRawTypes.get(RawTypes.Cpx), svDiscoveryInputData);
     }
 
+    private static void forNonComplexVariants(final EnumMap<RawTypes, JavaRDD<AssemblyContigWithFineTunedAlignments>> contigsByPossibleRawTypes,
+                                              final SvDiscoveryInputData svDiscoveryInputData) {
+
+        final String sampleId = svDiscoveryInputData.sampleId;
+        final Broadcast<ReferenceMultiSource> referenceBroadcast = svDiscoveryInputData.referenceBroadcast;
+        final Broadcast<SAMSequenceDictionary> referenceSequenceDictionaryBroadcast = svDiscoveryInputData.referenceSequenceDictionaryBroadcast;
+        final Broadcast<SVIntervalTree<VariantContext>> cnvCallsBroadcast = svDiscoveryInputData.cnvCallsBroadcast;
+        final String outputDir = svDiscoveryInputData.outputPath;
+
+        svDiscoveryInputData.updateOutputPath(outputDir+"/nonComplex.vcf");
```

Comment:
spaces around ```+``` if you modify this file for some other reason.

Reply:
done